db/index.js

```diff
@@ -4,6 +4,9 @@ import { dirname, join } from 'node:path'
 import { fileURLToPath } from 'node:url'
 import Postgrator from 'postgrator'
 
+/** @typedef {import('./typings.js').PgPools} PgPools */
```

Comment:
The convention is to use PascalCase for type names, i.e. start with a capital letter. I renamed `pgPools` type to `PgPools`.